src/test/fuzz/fuzz.cpp

```diff
@@ -77,6 +77,32 @@ void FuzzFrameworkRegisterTarget(std::string_view name, TypeTestOneInput target,
 
 static std::string_view g_fuzz_target;
 static const TypeTestOneInput* g_test_one_input{nullptr};
+bool ever_used_g_prng{false};
```

Comment:
nit: Naming + `static`
```suggestion
static bool g_ever_used_prng{false};
```